proprietary/tokens/src/components/ams/dialog.tokens.json

```diff
@@ -1,7 +1,7 @@
 {
   "ams": {
     "dialog": {
-      "background-color": { "value": "{ams.color.primary-white}" },
+      "background-color": { "value": "{ams.color.background.canvas}" },
```

Comment:
This should be `ams.color.background.default`, I think? In applications it's not supposed to be grey, but white